src/co/instea/BasicCommands_01.java

```diff
@@ -0,0 +1,24 @@
+package co.instea;
+
+import java.util.Scanner;
+
+public class BasicCommands_01 {
+    public static void main(String[] args) {
+        System.out.println("Enter following commands: ");
+        System.out.println("* 'h' for help");
+        System.out.println("* 'q' to quit the program");
```

Comment:
I would move list of commands to output of help command. Here I would leave only intro message with mention of _help_ command (for example 'Welcome to my console app, type 'h' to learn more.')